docs/x509.rst

```diff
@@ -181,6 +181,71 @@ X.509 Certificate Object
 
         For version 3 X.509 certificates.
 
+.. class:: Attribute
+
+    .. versionadded:: 0.8
+
+    .. attribute:: oid
+
+        :type: :class:`ObjectIdentifier`
+
+        The attribute OID.
+
+    .. attribute:: value
+
+        :type: :class:`str`
+
+        The value of the attribute.
+
+.. class:: ObjectIdentifier
+
+    .. versionadded:: 0.8
+
+    .. attribute:: value
+
+        :type: :class:`str`
```

Comment:
This will never contain unicode so calling it a str is safe for both py2 and py3. We are doing this for the `name` attributes on things like CipherAlgorithm and Mode already.